src/services/cronJobs/syncUserPowers.ts

```diff
@@ -63,13 +64,23 @@ export const getPreviousGivbackRoundInfo = (): {
   previousGivbackRound: number;
   fromTimestamp: number;
   toTimestamp: number;
+} => {
```

Comment:
There is something wrong here I guess. @mohammadranjbarz can we use the same code of getPowerBoostingSnapshotRound function in src/repositories/powerBoostingRepository.ts

Reply:
Done